EasyPost.NetFramework/ShipmentCollection.cs

```diff
@@ -17,7 +17,7 @@ public class ShipmentList : Resource
         ///     Get the next page of shipments based on the original parameters passed to Shipment.All().
         /// </summary>
         /// <returns>A new EasyPost.ShipmentList instance.</returns>
```

Comment:
update the comment

Reply:
Resolved in [73a3961](https://github.com/EasyPost/easypost-csharp/pull/223/commits/73a39618c0e40963cd3dd1b750ef53a13e56f3d1)